### ISSUE TITLE
Markdown parsing error

### ISSUE THREAD

**Issue by GNITOAHC** [TypeCellOS/BlockNote#518]
**Describe the bug**
Basically, the markdown to block conversion first converts the markdown to HTML using rehype/remark, then parses that HTML into ProseMirror nodes and converts those to blocks. Seems like the first step is going wrong, because the following markdown:
```
![image info](./img.png)
```
Gets converted into the following HTML:
```
<p><img src="./img.png" alt="image info"></p>
```

So the p tag wrapping is causing it to get parsed as a paragraph.

**To Reproduce**
The official blocknote [demo](https://www.blocknotejs.org/docs/converting-blocks#parsing-markdown-to-blocks)

**Comment by haoleexrex** [comment]
It also happens when use `tryParseMarkdownToBlocks`

**Comment by haoleexrex** [comment]
anyone know how to fix it? I really need it ><

**Comment by chinmay0703** [comment]
const parsedBlocks = await editor.tryParseMarkdownToBlocks(parsedata);   after parsing using tryParseMarkdownToBlocks image is not showing though the url pointing to the image

**Comment by matthewlipski** [comment]
The root cause of this is issue is more that a paragraph with only an image inside gets parsed as a paragraph rather than as an image on the ProseMirror side of things, which leads to issues parsing external content too (see #547). Anyway, should be fixed as of 0.14.0 now that #798 is merged.
